code/model/MultiFormStep.php

```diff
@@ -410,4 +417,43 @@ public function removeExtraClass($class) {
 	public function getExtraClasses() {
 		return join(' ', array_keys($this->extraClasses));
 	}
+
+	/**
+	 * Returns the submitted value, if any, of any steps.
+	 *
+	 * @todo  to be removed once https://github.com/silverstripe/silverstripe-multiform/pull/55 is merged in
+	 *
+	 * @param string $fromStep
+	 * @param string $key
+	 * @return mixed
+	 */
+	public function getValueFromOtherStep($fromStep, $key) {
+		// load the steps in the cache, if this one doesn't exist
+		if (!array_key_exists('steps_' . $fromStep, $this->cache)) {
+			$steps = MultiFormStep::get()->filter('SessionID', $this->form->session->ID);
+
+			if($steps) foreach($steps as $step) {
```

Comment:
Hello @tractorcow 

I fixed this to use ClassName instead. Makes sense to use the classname instead of the object. Silly me. Have a look and let me know what you think.

Cheers,
Spek